examples/cuda_umpire.cpp

```diff
@@ -0,0 +1,118 @@
+//#if defined(MFEM_USE_UMPIRE) && (defined(MFEM_USE_CUDA) || defined(MFEM_USE_HIP))
+
+#include "mfem.hpp"
+#include <unistd.h>
+#include <stdio.h>
+#include "umpire/Umpire.hpp"
+
+#define REQUIRE(a) if (!(a)) { printf("check failed: %s\n", #a); }
+
+
+using namespace mfem;
+
+constexpr unsigned num_elems = 1024;
+constexpr unsigned num_bytes = num_elems * sizeof(double);
+constexpr double host_val = 1.0;
+constexpr double dev_val = 1.0;
+
+static long alloc_size(int id)
+{
+   auto &rm = umpire::ResourceManager::getInstance();
+   auto a                    = rm.getAllocator(id);
+   return a.getCurrentSize();
+}
+
+int main(int argc, char ** argv)
```

Comment:
I think we need friendlier example here with more docs.

Reply:
This isn't going to stay as an example (going to be a test) but it was easier to have it here when running repeatedly (for interface updates)